### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,67 @@
...
+Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/). Click on the **Components** subtab and click **Create component**. Select the `base` type, then select the `agilex-limo` model. Enter a name for your base and click **Create**.
 
-```javascript
+On the new component panel, copy and paste the following attribute template into your base’s **Attributes** box. 
+```json
 {
-  "drive_mode": "", //Required. valid values are "differential", "ackermann", or "omni"
-  "serial_path": "" //Optional. serial path for communication. Defaults to "/dev/ttyTHS1"
+  "drive_mode": "<ackermann|differential|omni>",
+  "serial_path": "<your-serial-path>"
```
Comment (stevebriskin) [inline README.md:20]: this is now optional. (always has been, but was incorrectly defined).

File: README.md [hunk 2]
```diff
@@ -1,34 +1,67 @@
...
+
+### Attributes
+
+The following attributes are available for `viam:base:agilex-limo` bases:
 
-```javascript
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `drive_mode` | string | **Required** | LIMO [steering mode](https://docs.trossenrobotics.com/agilex_limo_docs/operation/steering_modes.html#switching-steering-modes). Options: `differential`, `ackermann`, `omni` (mecanum). |
+| `serial_path` | string | **Required** | The full filesystem path to the serial device, starting with <file>/dev/</file>. With your serial device connected, you can run `sudo dmesg \| grep tty` to show relevant device connection log messages, and then match the returned device name, such as `ttyTHS1`, to its device file, such as <file>/dev/ttyTHS1</file>. If you omit this attribute, Viam will attempt to automatically detect the path.<br>Default: `/dev/ttyTHS1` |
```
Comment (stevebriskin) [inline README.md]: same. Optional. Also "If you omit this attribute, Viam will attempt to automatically detect the path." isn't right, we'll just default to a value without "auto detecting".

**Review by stevebriskin — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (npentrel) [inline README.md]:
```suggestion
| `serial_path` | string | Optional | The full filesystem path to the serial device, starting with <file>/dev/</file>. With your serial device connected, you can run `sudo dmesg \| grep tty` to show relevant device connection log messages, and then match the returned device name, such as `ttyTHS1`, to its device file, such as <file>/dev/ttyTHS1</file>. If you omit this attribute, Viam will attempt to automatically detect the path.<br>Default: `/dev/ttyTHS1` |
```

File: README.md [hunk 3]
```diff
@@ -1,34 +1,67 @@
...
+
+The following attributes are available for `viam:base:agilex-limo` bases:
 
-```javascript
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `drive_mode` | string | **Required** | LIMO [steering mode](https://docs.trossenrobotics.com/agilex_limo_docs/operation/steering_modes.html#switching-steering-modes). Options: `differential`, `ackermann`, `omni` (mecanum). |
+| `serial_path` | string | Optional | The full filesystem path to the serial device, starting with <file>/dev/</file>. With your serial device connected, you can run `sudo dmesg \| grep tty` to show relevant device connection log messages, and then match the returned device name, such as `ttyTHS1`, to its device file, such as <file>/dev/ttyTHS1</file>. If you omit this attribute, Viam will attempt to automatically detect the path.<br>Default: `/dev/ttyTHS1` |
+
+### Examples configurations:
```
Comment (sguequierre) [inline README.md]:
```suggestion
### Example configurations:
```

**Review by sguequierre — CHANGES_REQUESTED** [review]

**Review by sguequierre — APPROVED** [review]

**Pull request merged**